yapapi/props/base.py

```diff
@@ -57,13 +58,13 @@ def get_type_args(t):
             else:
                 return getattr(t, "__args__")
 
-        # print("name=", self.name, "type=", self.type, type(self.type))
+        # print("name=", self.name, "type=", self.type, type(self.type), "value=", value)
```

Comment:
Could we remove this `print` altogether?

Reply:
well, I did find it useful when implementing this thing ... but sure, we can remove it as indeed we don't keep such debug lines in the code elsewhere...

Reply:
@azawlocki removed